config.xml

```diff
@@ -86,4 +86,5 @@
     <plugin name="cordova-plugin-sslcertificatechecker" spec="^5.1.0" />
     <engine name="ios" spec="4.5.4" />
     <engine name="android" spec="~7.0.0" />
+    <plugin name="cordova-plugin-aerogear-security" spec="../packages/security-cordova" />
```

Comment:
Good for now. We should release this plugin after we are content that it works as expected.